### PR TITLE
Add Aerobatics page

### PR DESCRIPTION
**Pull request by Hwurzburg**
needs to have the Tricks on a Switch (ie aerobatics in any mode) PR merged, of course
I intend to add to the video on how to clean up the LUA after creating a new trick from a template also

### CODE REVIEW

File: common/source/docs/common-scripting-aerobatics.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+
+.. note:: this capability is in ArduPlane 4.2 and higher
+
+ArduPilot has the capability of executing aerobatics from a LUA script. Either via AUTO mode mission items, or via scripts executed in many normal plane flight modes, usually controlled via a transmitter switch and selection RC channels.
+
+Mission NAV_SCRIPT_TIME 
+=======================
+
+The SCRIPT_TIME waypoint type allows for a :ref:`LUA<common-lua-scripts>` script to take control of the roll/pitch/yaw rates of the vehicle for a time. For protection against lack of an appropriate script being active (ie was not loaded in the APM/Scripts directory of the SD card) to assume control, or LUA script crash due to error, there is a mandatory parameter which provides a maximum time the script can takeover, before the next mission item is executed.
```
Comment (kd0aij) [inline common/source/docs/common-scripting-aerobatics.rst:15]:
```suggestion
The SCRIPT_TIME waypoint type allows for a :ref:`LUA<common-lua-scripts>` script to take control of the roll/pitch/yaw rates of the vehicle.  The mandatory timeout parameter is for protection against the lack of an appropriate script being active (ie was not loaded in the APM/Scripts directory of the SD card), or LUA script crash due to error.  The timeout value specifies the maximum time the script can take over before the next mission item is executed.
```

Comment (Hwurzburg) [reply common/source/docs/common-scripting-aerobatics.rst:15]: thanks

File: common/source/docs/common-scripting-aerobatics.rst [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+   <td><strong>param7</strong></td>
+   <td>Alt</td>
+   <td>Target altitude</td>
+   </tr>
+   </tbody>
+   </table>
+
+A LUA script would need to be running when the above mission command is encountered to recognize that it needs to take control, and start sending pitch/roll/yaw rate inputs into the flight controller to control its attitude and/or trajectory. It can use the two variable input parameters of the command to control aspects of the maneuver, such as rate of rolls/loops, repeat counts, time for certain segments of the maneuver, etc. An example is provided in the LUA Scripting Examples directory called `plane_aerobatics.lua </home/henry/Desktop/ardupilot/libraries/AP_Scripting/examples/Aerobatics/Missions>`__ for several types of tricks.
+
+The execution of a script running during the NAV_SCRIPT_TIME item can easily be interrupted by switching modes out of AUTO.
```
Comment (nkruzan) [inline common/source/docs/common-scripting-aerobatics.rst:69]: could you link to relevant page for NAV_SCRIPT_TIME

File: common/source/docs/common-scripting-aerobatics.rst [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+   </tr>
+   <tr style="color: #c0c0c0">
+   <td><strong>param7</strong></td>
+   <td>Alt</td>
+   <td>Target altitude</td>
+   </tr>
+   </tbody>
+   </table>
+
+A LUA script would need to be running when the above mission command is encountered to recognize that it needs to take control, and start sending pitch/roll/yaw rate inputs into the flight controller to control its attitude and/or trajectory. It can use the two variable input parameters of the command to control aspects of the maneuver, such as rate of rolls/loops, repeat counts, time for certain segments of the maneuver, etc. An example is provided in the LUA Scripting Examples directory called `plane_aerobatics.lua </home/henry/Desktop/ardupilot/libraries/AP_Scripting/examples/Aerobatics/Missions>`__ for several types of tricks.
```
Comment (nkruzan) [inline common/source/docs/common-scripting-aerobatics.rst:67]: bad link in here, /home/henry/blah

**Review by nkruzan — CHANGES_REQUESTED** [review]

File: common/source/docs/common-scripting-aerobatics.rst (hunk 3 above)
Comment (Hwurzburg) [reply common/source/docs/common-scripting-aerobatics.rst:67]: Can't put correct link until my PR gets merged moving that script

**Comment by izuhide** [comment]
On a fixed wing UAV, when I loaded plane_aerobatics.lua in MissionPlanner and entered ID number 14 in the flight plan using autopilot, it became a Rolling Circle (Figure Eight) instead of Scale Figure Eight. The result was not Scale Figure Eight, but Rolling Circle (Figure Eight). I was surprised by the sudden roll, but the plane landed safely without crashing.

**Comment by Hwurzburg** [comment]
do you have a log?

**Pull request merged**